manifests/04-deployment.yaml

```diff
@@ -41,6 +41,11 @@ spec:
           value: "6443"
         - name: KUBERNETES_SERVICE_HOST
           value: "127.0.0.1"
+        - name: METRICS_PORT
+          value: "9090"
```

Comment:
Not a hard call but 9090 is the usual port for the Prometheus server itself so it feels a bit confusing here.